ksqldb-execution/src/main/java/io/confluent/ksql/exception/KafkaTopicExistsException.java

```diff
@@ -17,8 +17,19 @@
 
 public class KafkaTopicExistsException extends KafkaTopicClientException {
 
-  public KafkaTopicExistsException(final String message) {
+  private final boolean partitionOrReplicaMismatch;
+
+  public KafkaTopicExistsException(
+      final String message,
+      final boolean partitionOrReplicaMismatch
```

Comment:
The value of this field is currently true in all cases, but given the generic name `KafkaTopicExistsException` I thought it'd be less confusing to add the extra field than assume that all instances of `KafkaTopicExistsException`s represent partition or replication factor mismatches.